src/RoboFile/ClientContainer.php

```diff
@@ -42,7 +42,24 @@ public function runContainerTests(
 		$opts['install-test'] = '';
 
 		$this->runContainerTestPreparation($opts);
+
 		$this->runTestSuites($opts);
+
+	}
+
+	public function runContainerTest(
```

Comment:
I think they may have two separate usecases, as one also runs the ContainerTestPreparation and a whole tests suite, while the other runs just one test.